.ci.yaml

```diff
@@ -2388,6 +2388,16 @@ targets:
         ["devicelab", "android", "linux"]
       task_name: list_text_layout_impeller_perf__e2e_summary
 
+  - name: Linux_android native_assets_android
+    recipe: devicelab/devicelab_drone
+    presubmit: false
```

Comment:
Out of curiosity why should these tests not be run in presubmit? Are they too slow? unlikely to break? something else?

Reply:
I think I might have copypasted that from the native assets tests for the other OSes.

@vashworth @stuartmorgan Could this be a remnant of the bringup strategy? Should we run these in presubmit?